lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1966,8 +1966,16 @@
       "target": "_blank",
       "devtoolsNodePath": "3,HTML,1,BODY,36,A",
       "selector": "body > a",
+      "boundingRect": {
+        "top": 1166,
+        "bottom": 1184,
+        "left": 91,
+        "right": 171,
+        "width": 80,
+        "height": 18
+      },
+      "snippet": "<a href=\"https://www.google.com/\" target=\"_blank\">",
       "nodeLabel": "external link",
-      "outerHTML": "<a href=\"https://www.google.com/\" target=\"_blank\">",
```

Comment:
outerHTML is removed on the artifact, its value was the same as the snippet value